pkg/util/singularityconf/config.go

```diff
@@ -42,6 +42,8 @@ type File struct {
 	CniConfPath             string   `directive:"cni configuration path"`
 	CniPluginPath           string   `directive:"cni plugin path"`
 	MksquashfsPath          string   `directive:"mksquashfs path"`
+	MksquashfsProcs         string   `directive:"mksquashfs procs"`
```

Comment:
a `uint` instead would take care of the conversion and in a first time ensure that the value is greater or equal to 0

Reply:
Good point! I just added a commit to convert MksquashfsProcs to uint.